backend/src/app.ts

```diff
@@ -19,7 +20,10 @@ app.use(requestLogger);
 
 app.use('/api/health', healthRouter);
 app.use('/api/user', userRouter);
-app.use(express.static('./public/'));
+// Static files are served by NGINX in production
```

Comment:
Is it a problem if we have this also in production? If not I would suggest leaving this as it was since this is an open source project and someone might use this in different environment.